app/repositories/ItemRepository.java

```diff
@@ -0,0 +1,42 @@
+package repositories;
+
+import com.amazonaws.services.dynamodbv2.xspec.S;
+import models.Item;
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.criterion.Restrictions;
+import play.db.jpa.JPAApi;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+
+@Singleton
+public class ItemRepository {
+
+    @Inject private JPAApi api;
+
+    public void createItem(Item item){
+        api.em().persist(item);
+    }
+
+    public void update(Item item){
+        api.em().unwrap(Session.class).update(item);
+    }
+
+    public List<Item> getAllItems(){
+        List<Item> list = api.em().unwrap(Session.class).createCriteria(Item.class).list();
```

Comment:
Obzirom da se
```java
 api.em().unwrap(Session.class)
```
ponavlja više puta, bilo bi pametno izdvojiti ga u posebnu metodu.
Generalno, treba obratiti pažnju da se piše kod koje je reuse-able, u smislu da se piše na način da se može uprotijebiti na što više mjesta moguće